src/pages/components/anchor.mdx

```diff
@@ -63,6 +63,17 @@ import AnchorExternalCode from '!!raw-loader!../../examples/anchor/AnchorExterna
   <AnchorExternal />
 </CodeExample>
 
+#### Security
```

Comment:
Positioning and hierarchy is probably not ideal. We need to come up with a rule for how/where this kind of content lives. Is it just paragraph text under the component heading (under `###Anchor` below)? Or should this be removed and the `isExternal` prop description amplified under `react-components`? (My opinion is that it makes more sense as a prop description.)